src/libutil/filesystem.cpp

```diff
@@ -20,7 +20,12 @@
 #include <OpenImageIO/ustring.h>
 
 #ifdef _WIN32
-// # include <windows.h>   // Already done by platform.h
+#    define WIN32_LEAN_AND_MEAN
+#    define VC_EXTRALEAN
+#    define NOMINMAX
+// clang-format off
+#    include <windows.h>
+// clang-format on
```

Comment:
```suggestion
#    include <windows.h>

```

I don't think you need the clang-format on/off, a blank line after the windows.h include will prevent it from being sorted with the subsequent lines.

Reply:
Ah cool! Did not know about this trick. Amended the commit with the change.